docs/commit-format.md

```
@@ -0,0 +1,19 @@
+# Commit Message Format
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)